cdc/scheduler/internal/v3/agent/table.go

```diff
@@ -79,8 +79,11 @@ func newAddTableResponseMessage(status tablepb.TableStatus) *schedulepb.Message
 		DispatchTableResponse: &schedulepb.DispatchTableResponse{
 			Response: &schedulepb.DispatchTableResponse_AddTable{
 				AddTable: &schedulepb.AddTableResponse{
-					Status:     &status,
-					Checkpoint: status.Checkpoint,
+					Status: &status,
+					Checkpoint: tablepb.Checkpoint{
+						ResolvedTs:   status.Checkpoint.CheckpointTs,
```

Comment:
why is the resolved ts assigned with the value  checkpoints

Reply:
Because resolvedTs should equal to checkpointTs when removing table.